lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/Identifiable.java

```diff
@@ -18,6 +18,9 @@
  */
 package org.apache.reef.wake;
 
+/**
+ * Identifier that can have id.
```

Comment:
This looks a bit weird; I'd use same wording as we have in `org.apache.reef.io.naming.Identifiable` - "This interface imposes that each object of the class that implements it has an identifier."